### ISSUE TITLE
Option seed_nodes does not work for a single node

### ISSUE THREAD

**Issue by Lanzaa** [mkjellman/perlcassa#31]
If there is only one node in the cluster the seed_nodes option does not work.

This may be related to the fact that the system.peers table does not contain the node that runs the query.

``` perl
my $obj = new perlcassa(
    'columnfamily' => 'myCF',
    'keyspace' => 'myKeyspace',
    'seed_nodes' => ['host1.cassandra.local'],
);
```

**Comment by swatidhoke** [comment]
Can you please help me with issue 30 of long data type not supported.

**Comment by mkjellman** [comment]
@swatidhoke yes, now sure how your comment is related to this issue. Why are you cross posting here?

**Comment by swatidhoke** [comment]
oops my bad , please ignore above comment. didn't meant to be posted under this thread.

**Comment by swatidhoke** [comment]
Swati Dhoke sent you an invitation

Twitter helps you stay connected with what's happening right now and with the people and organizations you care about.

```
Accept invitation
```

https://twitter.com/i/3b0eb9ef-4507-4230-b5f1-24c642dbde4b

## 

This message was sent by Twitter on behalf of Twitter users who entered your email address to invite you to Twitter.
Unsubscribe: https://twitter.com/i/o?t=1&iid=6228e3131f214a2885966690663a3988&uid=0&c=IuVwbCB3dgFj4ra68zKho2nV%2B8sbS%2Bp%2FOmeNuVQOOFbvS7FnCbF%2BUjRKtlDvVn%2BDcZC%2FqsJ0hm4ETcT7Ur%2BjT0zr5Tfv20Qm2zq1BZwyz%2BJ4Jt159J7uXg%3D%3D&nid=9+26

Need help?
https://support.twitter.com
